docs/reference/upload-rights.rst

```diff
@@ -0,0 +1,120 @@
+Ubuntu kernel.
+
+- Members of the ubuntu-kernel-uploaders_ Launchpad team
+
+- Generally have commit access to the `Ubuntu kernel git repositories`_
+
+- Actively follow and participate in discussions and patch reviews on the
+  `Ubuntu kernel-team mailing list`_
+
+- Are collectively responsible for the maintenance of packages in the Ubuntu
+  kernel package set for all supported releases as well as the development
+  release
+
+- Have a strong working knowledge of kernel packaging concepts and techniques,
+  refined through experience
+
+- Have a strong working knowledge of Ubuntu project procedures, especially
+  those related to the release process and support commitments, and an
+  understanding of the reasons why they exist
+
+- Have a history of substantial direct contributions to the distribution and
+  particularly the kernel related packages
+
+- Feel a sense of personal responsibility for the quality of Ubuntu releases
+  and for the satisfaction of Ubuntu users
+
+- Exercise great care in their work, with the understanding that their efforts
+  have a direct impact on others, including:
+
+  - every Ubuntu user
+  - the Ubuntu release team
+  - corporate partners who provide support for Ubuntu
+
+Application Process
+===================
+
+As alluded to in the member profile above, membership consideration for the
+ubuntu-kernel-uploaders team adheres to a strict policy. Anyone considering
+applying should meet **all** of the criteria listed below:
+
+1. Have a history of substantial and direct contribution to the distribution;
+   particularly the kernel related packages
+
+2. A thorough understanding of the Ubuntu kernel patch submission process
+
+   a. Demonstrates an understanding of this process by having submitted
+      multiple patches which were accepted over a six-month development cycle
+   b. Demonstrates an understanding of this process by having reviewed and
+      Ack'd/Nack'd multiple patches over a six-month development cycle 
+
+3. A thorough understanding of the Ubuntu release cycle and associated
+   milestone and freeze dates
+
+4. A thorough understanding of the Ubuntu Kernel SRU cadence
+
+5. A thorough understanding of the upstream kernel development cycle and how it
+   relates to the Ubuntu kernel development cycle
+
+6. Demonstrate a chain of trust by having multiple sponsored kernel uploads
+   over a six-month development cycle by various existing members of the
+   ubuntu-kernel-uploaders team.
+
+If you are not an official ubuntu-kernel-uploaders member yet, but fulfill all
+of the criteria above, you are likely a promising candidate for joining the
+team. If you are interested in joining, please start by preparing your
+application. Please use the following template for your application:
+
+https://wiki.ubuntu.com/Kernel/Dev/PPUApplicationTemplate
+
+An example application can also be seen at the following: 
+
+https://wiki.ubuntu.com/LuisHenriques/PerPackageUploaderApplication 
```

Comment:
We should consider including the template and sample application somewhere in this repo also.

Reply:
I misunderstood the purpose of the page and template where the application process requires it to be in the applicant's Wiki.
These should be left as is.

Thanks!